src/store/Reducers/PatchSetsReducer.js

```diff
@@ -0,0 +1,53 @@
+import * as ActionTypes from '../ActionTypes';
+import {
+    changeFilters,
+    selectRows,
+    fetchPending,
+    fetchRejected
+} from './HelperReducers';
+
+// Initial State. It should not include page and perPage to persist them dynamically
+export const initialState = {
```

Comment:
`Object.freeze` only does a shallow freeze, meaning it only freezes the immediate properties, but not properties of objects assigned to the properties. For example, while it would not be possible to do `initialState. queryParams = ...` it is possible to do `initialState.queryParams.page = 1`.